QENSmodels/gaussian.py

```diff
@@ -76,16 +76,36 @@ def gaussian(x, scale=1., center=0., sigma=1.):
       | ``sigma``    | Sigma                                  |
       +--------------+----------------------------------------+
 
+    * Numerical issues:
+      The definition of the Lorentzian function used here is such that
+      the integral between :math:`-\infty` and :math:`-\infty` is 1.
+      However, when the hwhm is comparable or smaller than the x step,
+      the sampling of the function will result in a numerical integral > 1.
+      E.g., in the extreme case where hwhm tends to zero, the numerical
+      sampling at x points will result in a delta-like function, but with
+      a value at maximum approaching :math:`-\infty` instead of the
+      value 1/:math:`\Delta x` used in the definition of the delta function.
+      Therefore, the value of the integral of the function is checked
+      and used to renormalize the returned function whenever the integral
+      is larger than 1.
 
     """
     x = np.asarray(x)
 
     if sigma == 0:
         model = QENSmodels.delta(x, scale, center)
     else:
-        model = scale / (sigma * np.sqrt(2. * np.pi)) \
+        model = (sigma * np.sqrt(2. * np.pi)) \
             * np.exp(- (x - center) ** 2 / (2. * sigma ** 2))
 
+    # Area normalization
+    area = np.trapz(model, x)
+    if area > 1:
+        model /= area
+
+    # Scale by amplitude
+    model *= scale
```

Comment:
If `hwhm`=0,` scale` is applied twice since it is already in the expression of `delta`.

Reply:
Yes, I'll correct this.